src/OnlineRetailPortal.Contracts/Models/GetProductServiceRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Contracts
+{
+  public class GetProductServiceRequest
+    {
```

Comment:
Class Name should be ProductServiceRequest. Any method names which will fetch this should data should be named as GetProductServiceRequest